File: Sis_Gerenciamento_Invet/src/avaliação/Product.java
package avaliação;

public abstract class Product {
    String name;
    double price;
    String SKU;

    public Product(String name, double price, String SKU) {
        this.name = name;
        this.price = price;
        this.SKU = SKU;
    }
    
    public abstract void displayInfo();
}

File: Prova_27.11/ex2/models/Product.js
module.exports = (sequelize, DataTypes) => {
    const Product = sequelize.define('Product', {
      id: { type: DataTypes.STRING, primaryKey: true },
      name: { type: DataTypes.STRING, allowNull: false },
      description: { type: DataTypes.STRING },
      price: { type: DataTypes.DOUBLE },
    });
    return Product;
  };
  